src/config/constants.js

```diff
@@ -50,6 +50,7 @@ export const ENDPOINT = {
   AUTH_CHANGE_DISPLAY_VALUE: '/authority-record/fixed-field-display-value',
   BIBLIOGRAPHIC_RECORD: '/bibliographic-record',
   AUTHORITY_RECORD: '/authority-record',
+  AUTHORITY_RECORD_DETAILS: 'authority-record-details',
```

Comment:
Shouldn't this have a `/` at the beginning?

Reply:
You are right. I have fixed it